### PR TITLE
Add links to slide decks from core and elective courses

### PR DESCRIPTION
**Pull request by terichadbourne**
This PR:
-  adds links to PDF slides for Core Courses A (cc @alanshaw) and D (cc @olizilla) within their own folders and readmes
- adds links to known published slides for all core and elective slides to the table in main readme for camp

@olizilla You mentioned wanting to add a version with speaker notes in addition to the PDF, but I don't have time today to figure out how to do that without making your deck editable. Please feel free to add a second link following the example in Core Course D. 

@jacobheun & @lidel & @NukeManDan this further promotes the slides you added in previous PRs by surfacing on the main page

I'll be on vacation next week and don't have admin access to this repo regardless, so please feel free to make edits or go ahead and merge if all looks good. 

This partially addresses issue #108, but I've only tracked down slides for core courses, not electives. Perhaps @autonome or @olizilla would know which presenter was the lead for each of those to request them?

### CODE REVIEW

**Comment by olizilla** [comment]
I've updated the google slides sharing settings for core course D to "anyone with the link can view"... I think it's super useful to be able to read the speaker notes, as it's a complete transcript of what I said in the course. It's great to also have the PDF so the user can choose which flavour they want.

I'm in favour of sharing both flavours on all courses, unless there is a good reason not to.

**Comment by terichadbourne** [comment]
I've added a link to the Elective A slides based on @satazor's separate PR #181 which adds them to his own course's homepage. 

Both of these PRs need merging but I'm unable to see who has merging powers on this repo.

File: README.md [hunk 1]
```diff
@@ -15,22 +15,22 @@ You can find the content description, speakers and trainers at:
 
 ## [👩🏽‍🏫 Core & Elective Courses](CORE_AND_ELECTIVE_COURSES)
 
-| Title  | Trainers  | Short Description  | Materials  | Recording |
+| Course | Title  | Trainers  | Short Description  | Materials  | Video Recording |
```
Comment (alanshaw) [inline README.md:18]: This table does not render good anymore :(

Screenshot 2019-09-04 at 16 16 21

**Review by alanshaw — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (terichadbourne) [reply README.md:18]: Thanks for the catch! I believe I've successfully fixed this!

**Comment by terichadbourne** [comment]
Fixed the table formatting error discovered by @alanshaw and added @olizilla's new Google Slides link to the main readme page table. Good to go?

**Pull request merged**